### PR TITLE
Merge master into staging ln

### PR DESCRIPTION
**Pull request by Beerosagos**
[Merge branch 'master' into merge-master-staging-ln](https://github.com/digitalbitbox/bitbox-wallet-app/pull/2687/commits/81a31ce281f7c9c49b7a7dc9be2c9c000d1db6f2)

Conflicts:
	backend/backend.go
	backend/devices/bitbox02bootloader/assets/firmware-btc.sha256
	backend/devices/bitbox02bootloader/assets/firmware.sha256
	backend/devices/bitbox02bootloader/firmware.go
	backend/handlers/handlers.go
	frontends/web/src/components/sidebar/sidebar.tsx
	frontends/web/src/locales/en/app.json
	frontends/web/src/routes/account/summary/accountssummary.tsx
	frontends/web/src/routes/router.tsx
	frontends/web/src/routes/settings/advanced-settings.tsx
	go.mod
	go.sum
	vendor/modules.txt

### CODE REVIEW

File: backend/handlers/handlers.go [hunk 1]
```diff
@@ -763,6 +764,59 @@ func (handlers *Handlers) getAccountsBalance(*http.Request) (interface{}, error)
 	return totalAmount, nil
 }
 
+// getCoinsTotalBalance returns the total balances grouped by coins.
+func (handlers *Handlers) getCoinsTotalBalance(_ *http.Request) (interface{}, error) {
+	totalPerCoin := make(map[coin.Code]*big.Int)
+	conversionsPerCoin := make(map[coin.Code]map[string]string)
+
+	totalAmount := make(map[coin.Code]coin.FormattedAmount)
```
Comment (benma) [inline backend/handlers/handlers.go:772]: not a rebase error but this should be `coinpkg.FormattedAmount` - mixing imports.

**Review by benma — APPROVED** [review]
LGTM. that rebase was horrific - I wonder if we can do sth to make that easier. Lots of conflicts that needed a lot of concentration to figure out.

File: backend/handlers/handlers.go (hunk 1 above)
Comment (Beerosagos) [reply backend/handlers/handlers.go:772]: All the other FormattedAmount occurrences in the file are `coin.FormattedAmount`, I just thought to keep the same. Anyway I guess we should remove all the `coin.Something` calls and substitute them with `coinPkg.` to remove the double import.

**Comment by Beerosagos** [comment]
Yeah that was a hard one. Even if we keep all the LN stuff in a separate package it still affects the code in a lot of different places.
Thanks for the review :pray:

**Pull request merged**